ratpack-bearer-auth/src/main/java/st/ratpack/auth/handler/TokenScopeFilterHandler.java

```diff
@@ -35,6 +35,6 @@ public void handle(Context ctx) throws Exception {
 
 	private void sendError(Context ctx) {
 		//Not authorized stop the chain here
-		ctx.getResponse().status(401).send();
+		ctx.getResponse().status(403).send();
```

Comment:
Looking at the usage above:
```
		if (oAuthTokenOptional.isPresent()) {
			OAuthToken token = oAuthTokenOptional.get();
			Sets.SetView<String> sharedScopes = Sets.intersection(token.getScope(), allowedScopes);
			if (!sharedScopes.isEmpty()) {
				ctx.next();
			} else {
				sendError(ctx);
			}
		} else {
			sendError(ctx);
		}
```

Would it make more sense to send a 401 when the token is not present and a 403 with token is present w/o correct scopes?